frontend/src/services/gameNightTrackerApi.js

```diff
@@ -4,3 +4,8 @@ export const getAllGames = (token) =>
     axios
         .get('user/games', { headers: { Authorization: 'Bearer ' + token } })
         .then((response) => response.data);
+
+export const getAllSessions = (token) =>
+    axios
+        .get('user/sessions', { headers: { Authorization: 'Bearer ' + token } })
```

Comment:
Think about configuring axios with your token so you don't have to repeat yourself on every request.